balancer/rls/picker_test.go

```diff
@@ -57,9 +57,9 @@ func (s) TestNoNonEmptyTargetsReturnsError(t *testing.T) {
 	r := startManualResolverWithConfig(t, rlsConfig)
 
 	// Dial the backend.
```

Comment:
Above comment `Dial the backend` should change to `Create new client.` Applicable everywhere

Reply:
done please check